src/javascript/app_2/Modules/Reports/Components/market-symbol-icon-row.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes          from 'prop-types';
+import React              from 'react';
+import { UnderlyingIcon } from 'App/Components/Elements/underlying-icon.jsx';
+import { IconTradeType }  from '../../../Assets/Trading/Types';
+
+const getMarketInformation = (payload) => {
+    const { shortcode } = payload;
+    const pattern = new RegExp('(^[A-Z]+)_((R_\\d{2,3})|[A-Z]+)'); // Used to get market name from shortcode
+    const extracted = pattern.exec(shortcode);
```

Comment:
we could use `payload.shortcode` as it doesn't seem to be used anywhere else in the function